src/lib/Scenes/Map/GlobalMap.tsx

```diff
@@ -242,17 +260,20 @@ export class GlobalMap extends React.Component<Props, State> {
   }
 
   render() {
-    const { city } = this.props.viewer
-    const { lat: centerLat, lng: centerLng } = this.props.initialCoordinates || city.coordinates
+    const city = get(this.props, "viewer.city")
+    const { lat: centerLat, lng: centerLng } = this.props.initialCoordinates || get(city, "coordinates")
+    const { mapLoaded } = this.state
 
+    // TODO: Need to hide the map while showing the top buttons.
```

Comment:
@l2succes we'll need to refactor the view hierarchy to hide the map while still showing the city picker button.